medicines/README.md

```diff
@@ -1,33 +1,26 @@
-# Medicines Information Portal
+# MHRA Products Portal
 
-![](./docs/architecture.svg)
+This project contains the source code for the medicines information portal, which can be found at: https://products.mhra.gov.uk/
```

Comment:
&hellip;and on this line it's back to _medicines information portal_. Which is correct?